test/functional/webauthn_verifications_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require "test_helper"
+
+class WebauthnVerificationsControllerTest < ActionController::TestCase
+  context "#verify" do
+    context "when given an expired webauthn token" do
+      setup do
+        @user = create(:user)
+        token = create(:webauthn_verification, user: @user, path_token_expires_at: 1.minute.ago).path_token
```

Comment:
Nit: this is a verification, not a token.

Reply:
Here it's the `verification.path_token`

Reply:
Ah! So it is.